app/renderer/actions/Inspector.js

```diff
@@ -377,11 +382,19 @@ export function findAndAssign (strategy, selector, variableName, isArray) {
   };
 }
 
-
-// TODO: Is this obsolete? I don't think we use this.
 export function setLocatorTestElement (elementId) {
```

Comment:
shouldn't this function be also defined as async?

Reply:
The function that is returned with is `async`.